tests/functional/object/mcg/test_namespace_crd.py

```diff
@@ -152,6 +156,7 @@ def deprecated_test_namespace_store_creation_crd(
                         "type": "Single",
                         "namespacestore_dict": {"aws": [(1, None)]},
                     },
+                    "nss_creation_deletion_timeout": 240,
```

Comment:
Please change timeout in TimeoutSampler to 240 to avoid adding timeout for each case